soh/soh/OTRGlobals.h

```diff
@@ -146,7 +146,8 @@ void EntranceTracker_SetLastEntranceOverride(s16 entranceIndex);
 void Gfx_RegisterBlendedTexture(const char* name, u8* mask, u8* replacement);
 void SaveManager_ThreadPoolWait();
 
-int32_t GetGIID(uint32_t itemID);
+GetItemID GetGetItemIDFromItemID(ItemID itemID);
```

Comment:
I like the idea of expanding this, but maybe, for clarity, we should start using "Retrieve" instead of "Get", as "GetGet" is bound to cause some confusion to anyone looking at this code for the first time.